docs/SPEC.md

```diff
@@ -93,35 +90,20 @@ Examples:
 
 The following keywords are reserved and may not be used as identifiers:
 
-    and    import  not  return   option   test
-    empty  in      or   package  builtin
-
-[IMPL#256](https://github.com/influxdata/flux/issues/764) Add in and empty operator support
+    and import  option   if
+    or  package builtin  then
+    not return  testcase else exists
 
 #### Operators
 
 The following character sequences represent operators:
 
     +   ==   !=   (   )   =>
     -   <    !~   [   ]   ^
-    *   >    =~   {   }
-    /   <=   =    ,   :
+    *   >    =~   {   }   ?
+    /   <=   =    ,   :   "
     %   >=   <-   .   |>
 
-#### Numeric literals
```

Comment:
We tried to implement numeric literals but were unsuccessful in making it complete through the entire implementation. We can revisit this later but for now its no longer part of the SPEC.

Reply:
`?` and `"` are operators?